README.md

```diff
@@ -0,0 +1,114 @@
+factorization](https://en.wikipedia.org/wiki/Non-negative_matrix_factorization) is used on each analyzed image to
+simultaneously derive the amount of hematoxylin and eosin stain at each pixel and the image-wide effective colors of
+each stain.
+
+The implementation in ITK accelerates the non-negative matrix factorization by choosing the initial estimate for the
+color absorption characteristics using a technique mimicking that presented in [Arora et al.,
+2013](http://proceedings.mlr.press/v28/arora13.html) and modified in [Newberg et al.,
+2018](https://doi.org/10.1371/journal.pone.0193067).  This approach finds a good solution for a non-negative matrix
+factorization by first transforming it to the problem of finding a convex hull for a set of points in a cloud.
+
+The lead developer of InsightSoftwareConsortium/ITKColorNormalization is [Lee Newberg](https://github.com/Leengit/).
+
+## Installation for Python
+
+ITKColorNormalization and all its dependencies can be easily installed with [Python
+wheels](https://blog.kitware.com/itk-is-on-pypi-pip-install-itk-is-here/).  Wheels have been generated for macOS, Linux,
+and Windows and several versions of Python, 3.5, 3.6, 3.7, and 3.8.  If you do not want the installation to be to your
+current Python environment, you should first create and activate a [Python virtual environment
+(venv)](https://docs.python.org/3/tutorial/venv.html) to work in.  Then, run the following from the command-line:
+
+    pip install itk-spcn
+
+Launch `python`, import the itk package, and set variable names for the input images
+
+    import itk
+    input_image_filename = 'path/to/image_to_be_normalized'
+    reference_image_filename = 'path/to/image_to_be_used_as_color_reference'
+
+## Usage in Python
+
+The following example transforms this input image
+
+![Input image to be normalized](https://data.kitware.com/api/v1/file/57718cc48d777f1ecd8a883f/download)
+
+using the color scheme of this reference image
+
+![Reference image for normalization](https://data.kitware.com/api/v1/file/576ad39b8d777f1ecd6702f2/download)
+
+to produce this output image
+
+![Output of spcn_filter](https://data.kitware.com/api/v1/file/5ed685d89014a6d84e9bc6f0/download)
+
+### Functional interface to ITK
+
+You can use the functional, eager interface to ITK to choose when each step will be executed as follows.  The
+`input_image` and `reference_image` are processed to produce `normalized_image`, which is the `input_image` with the
+color scheme of the `reference_image`.  The `color_index_suppressed_by_hematoxylin` and
+`color_index_suppressed_by_eosin` arguments are optional if the `input_image` pixel type is RGB or RGBA.  Here you are
+indicating that the color channel most suppressed by hematoxylin is 0 (which is red for RGB and RGBA pixels) and that
+the color most suppressed by eosin is 1 (which is green for RGB and RGBA pixels)\; these are the defaults for RGB and
+RGBA pixels.
+
```

Comment:
Add triple backticks here for code formatting.

Reply:
Done.